pkg/controller/install/openshift/openshift.go

```diff
@@ -86,6 +87,168 @@ func EnsureOpenshiftIngress(c client.Client, scheme *runtime.Scheme, instance *s
 	return nil
 }
 
+// EnsureConfigMapForCrt ensures crt injection into the configmap
+func EnsureConfigMapForCrt(c client.Client, scheme *runtime.Scheme, instance *servingv1alpha1.Install) error {
+	namespace := instance.GetNamespace()
+	if routeExists, err := kindExists(c, "route", "route.openshift.io/v1", namespace); err != nil {
+		return err
+	} else if !routeExists {
+		// Not running in OpenShift
+		return nil
+	}
```

Comment:
Checking for the route shouldn't be necessary anymore, which renders this function superfluous. You just need the `updateConfigMapForCrt` now.

Reply:
Okey...